docs/testplan/Chassis-fabric-test-plan.md

```diff
@@ -0,0 +1,66 @@
+# **VOQ Chassis Fabric Test Plan**
+
+ - [Introduction](#introduction)
+ - [Scope](#scope)
+ - [Assumptions](#assumptions)
+ - [Test Setup](#test-setup)
+ - [Test Cases](#test-cases)
+     
+# Introduction 
+
+This is the test plan for Fabric link testing on SONIC Distributed VOQ System, as described in the [VOQ Fabric HLD](https://github.com/Azure/SONiC/blob/master/doc/voq/fabric.md)
+
+## Scope
+
+The scope of this test plan is as follows:
+* Check if all the expected fabric links are up.
+* Check if the fabric counters work correctly when there is data traffic.
+* Check if the fabric devices can reach all of the forwarding ASICs of the chassis (reachability)
+
+## Assumptions
+
+The current SW design for fabric does not cover events like card insertion/removal or reboots. This test plan depends on fabric counter cli support (work in progress).
+
+# Test Setup
+
+These test cases will be run in the proposed [T2 topology](https://github.com/Azure/sonic-mgmt/blob/master/ansible/vars/topo_t2.yml). It is assumed that such a configuration is deployed on the chassis.
+
+tbinfo will be populated with the number of fabric links per forwarding ASIC that are expected to be up.
```

Comment:
tbinfo is derived from the testbed.yaml/testbed.csv - which is generic for all types of DUTs - including pizza boxes. This field seems to be specific to VoQ chassis - so recommend using the host_var from the inventory file, rather than tbinfo.

Reply:
Fixed in latest revision